src/fparser/two/Fortran2008/Fortran2008.py

```diff
@@ -1718,62 +1768,10 @@ def tostr(self):
         return f"{result} {self.items[0]}"
 
 
-#
-# GENERATE Scalar_, _List, _Name CLASSES
-#
-
-
-ClassType = type(Base)
-_names = dir()
-for clsname in _names:
-    new_cls = eval(clsname)
-    if not (
-        isinstance(new_cls, ClassType)
-        and issubclass(new_cls, Base)
-        and not new_cls.__name__.endswith("Base")
-    ):
-        continue
-
-    names = getattr(new_cls, "subclass_names", []) + getattr(new_cls, "use_names", [])
-    for n in names:
-        if n in _names:
-            continue
-        if n.endswith("_List"):
-            _names.append(n)
-            n = n[:-5]
-            # Generate 'list' class
-            exec(
-                f"""\
-class {n}_List(SequenceBase):
-    subclass_names = [\'{n}\']
-    use_names = []
-    @staticmethod
-    def match(string): return SequenceBase.match(r\',\', {n}, string)
-"""
-            )
-        elif n.endswith("_Name"):
-            _names.append(n)
-            n = n[:-5]
-            exec(
-                f"""\
-class {n}_Name(Base):
-    subclass_names = [\'Name\']
-"""
-            )
-        elif n.startswith("Scalar_"):
-            _names.append(n)
-            n = n[7:]
-            exec(
-                f"""\
-class Scalar_{n}(Base):
-    subclass_names = [\'{n}\']
-"""
-            )
-
-
 # Inspect the contents of this module and list all of the classes in __all__
```

Comment:
I think this bit needs moving to `__init__.py` too as it's only here because of the run-time generation of classes and these are no longer in this module (file). It probably needs a bit of finessing once moved so that it only adds the generated classes as opposed to all classes. That might be just a matter of excluding those that are imported but I don't know.

Reply:
OK, moved. It only adds the generated classes by default so we're OK, although I had to nullify the temporay `new_cls` variable as it was appearing in the `__all__` list, but this was an existing error.